src/styles/blocks/card.scss

```
@@ -0,0 +1,94 @@
+.card {
+  border: 1px solid $gray-elements-color;
+  border-radius: 5px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  font-family: $font-family;
+  font-size: $main-font-size;
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  &__img {
+    align-self: center;
+    height: 134px;
+    margin-bottom: 40px;
+    width: 160px;
+  }
+
+  &__title {
+    color: $main-accent-color;
+    font-size: 1em;
+    font-weight: 500;
+    line-height: 1.5em;
```

if you do it with `px` - do in all cases)
```suggestion
    line-height: px;
```